miners/text/template_miner.py

```diff
@@ -82,9 +82,9 @@ def add_args( parser: argparse.ArgumentParser ):
     def init_weights(self):
         initrange = 0.1
         self.remote_decoder.weight.data.uniform_(-initrange, initrange)
-        self.remote_decoder.bias.data.zero_()
+        #self.remote_decoder.bias.data.zero_()
```

Comment:
If we're not using this, let's just remove it altogether.

Reply:
Shows good, will remove